specs/swagger.yml

```diff
@@ -3060,28 +3060,46 @@ paths:
         '429':
           $ref: '#/responses/rateLimitedResponse'
   /database/check-data-integrity:
-    get:
-      summary: Runs an data integrity scan on the datastore and returns any found issues
-      description: Runs an data integrity scan on the datastore and returns any found issues. Requires admin access.
+    post:
+      summary: Starts a data integrity scan on the datastore
+      description: Starts a data integrity scan on the datastore. Requires admin access.
```

Comment:
I would mention what causes `429`. Same w/ `/database/fix-data-integrity` below. 

Do `check` and `fix` limit each other too? If so that is worth mentioning.

Example:

> Starts a data integrity scan on the data store. Requires admin access, can only run one instance.

Reply:
:+1:  Added more doc